src/validation.h

```diff
@@ -162,8 +162,13 @@ struct MempoolAcceptResult {
         VALID, //!> Fully validated, valid.
         INVALID, //!> Invalid.
         MEMPOOL_ENTRY, //!> Valid, transaction was already in the mempool.
+        WITNESS_SWAPPED, //!> A same-txid-different-witness tx already exists in the mempool, and was not replaced.
```

Comment:
Again, I think this terminology is a bit misleading, since swapped suggests the witness was replaced in the mempool.